CODEOWNERS

```diff
@@ -219,3 +219,4 @@ extensions/filters/http/oauth2 @rgs1 @derekargueta @snowp
 /contrib/cryptomb/ @rojkov @ipuustin
 /contrib/vcl/ @florincoras @rojkov
 /contrib/hyperscan/ @rojkov @zhxie
+/contrib/language/ @diceride @yanavlasov
```

Comment:
`contrib` extensions do not need maintainer sponsor. Please just leave your username.

Reply:
The CI is failing due to the following error

```
ERROR: Contrib extensions require at least 2 owners in CODEOWNERS:
```

It appears that it is not possible to only specify a single codeowner for a contrib extension. Do you know a way to disable this warning?